elastic/kibana
# [ResponseOps][Actions] Improve Task Manager’s retry logic for ad-hoc tasks

Resolves https://github.com/elastic/kibana/issues/143048

## Summary

Updated the retry logic in the task manager. If the first attempt encounters a failure we will retry 30 seconds later. If the second attempt fails, we will start a retry 5-minute multiple from the previous run. It will look like this

Attempt 1: now
Attempt 2: 30s after the first attempt
Attempt 3: 5m after the second attempt
Attempt 4: 10m after the third attempt
Attempt 5: 20m after the fourth attempt

### Checklist


### To verify

- Create a rule and then force a retry failure
- Verify that the retry follow the pattern set above ^
